README.rst

```diff
@@ -105,50 +89,33 @@ related online documentation:
 
+Xen support
+~~~~~~~~~~~
 
-Python bindings
-----------------
-LibVMI is written in C.  If you would rather work with Python, then look at the
-``libvmi/python``` repository. They provide an almost feature complete python
-interface to LibVMI with a relatively small performance overhead.
-
-https://github.com/libvmi/python
-
-Xen Support
------------
-If you would like LibVMI to work on Xen domains, you must simply ensure
-that you have Xen installed along with any Xen development packages.
-LibVMI should effectively just work on any recent version of Xen.
-
+LibVMI provides support for Xen out-of-the-box. If you install Xen from source,
+make sure the Xen libraries compiled from source are in your LD_LIBRARY_PATH. You don't
+have to recompile LibVMI if you update Xen as LibVMI is able to detect what version of Xen
+you have dynamically at runtime.
 
 XenServer Support
 -----------
 Compiling LibVMI on a XenServer dom0 can be challenging as there are no
 development headers and tools present. The recommended way to compile
 is in a separate CentOS installation using the Xen development packages
-matching what XenServer runs on. Make sure to remove the libvirt packages
-from CentOS (yum remove libvirt*) as to avoid the KVM driver getting
-activated in LibVMI. The compiled LibVMI library and tools can then be
-transferred to the XenServer dom0 and run natively.
+matching what XenServer runs on. The compiled LibVMI library and tools
+can then be transferred to the XenServer dom0 and run natively.
 
+KVM support
+~~~~~~~~~~~
+LibVMI will have KVM support if libvirt is available during compile time. Ensure that your
+libvirt installation supports QMP commands, most prepackaged versions do not support this
+by default so you may need to install libvirt from source yourself.  To enable QMP support
+when installing from source, ensure that you have libyajl-dev (or the equivalent from your
+linux distro) installed, then run the configure script from libvirt.  Ensure that the
+configure script reports that it found yajl.  Then run make && make install.
 
-KVM Support
------------
-If you would like LibVMI to work on KVM VM's, you must do some additional
-setup.  This is because KVM doesn't have much built-in capability for
-introspection.  For KVM support you need to do the following:
-
-- Ensure that you have libvirt version 0.8.7 or newer
-
-- Ensure that your libvirt installation supports QMP commands, most
-  prepackaged versions do not support this by default so you may need
-  to install libvirt from source yourself.  To enable QMP support
-  when installing from source, ensure that you have libyajl-dev (or
-  the equivalent from your linux distro) installed, then run the
-  configure script from libvirt.  Ensure that the configure script
-  reports that it found yajl.  Then run make && make install.
-
-- Choose a memory access technique:
+Currently there is no native VMI support in KVM, so you will have to use a custom version
```

Comment:
Could you rewrite this to say that you will only need to patch QEMU in order to get memory access for libvmi ?